etc/kayobe/inventory/groups

```diff
@@ -132,6 +132,9 @@ rgws
 [cis-hardening:children]
 overcloud
 
+[reboot_hosts:children]
+storage
+
```

Comment:
This is only used for multinodes. Move to ci-multinode env?

Reply:
Done